ui/pages/settings/settings.component.js

```diff
@@ -137,8 +137,8 @@ class SettingsPage extends PureComponent {
             {currentPath !== SETTINGS_ROUTE && (
               <ButtonIcon
                 ariaLabel={t('back')}
-                name={ICON_NAMES.ARROW_LEFT}
-                size={ICON_SIZES.XL}
+                iconName={ICON_NAMES.ARROW_LEFT}
+                size={ICON_SIZES.MD}
```

Comment:
LGTM! Just a suggestion, I think rather than changing the size we should restore this className to control its visibility on different screen size
```suggestion
                size={ICON_SIZES.MD}
                className="settings-page__back-button"
```

Reply:
There is no `ICON_SIZES.MD` make sure to check proptype errors :)